src/microbe_stage/MicrobeCamera.cs

```diff
@@ -80,6 +81,12 @@ public Vector3 CursorWorldPos
         private set => cursorWorldPos = value;
     }
 
+    public void ResetPosition()
+    {
+        ObjectToFollow.Translation = new Vector3(0, 0, 0);
+        ResetHeight();
+    }
+
```

Comment:
I did say you should put this in this class, but on second thought, this should be in the editor...
Because if someone calls this on the player camera in the microbe stage, it's going to ruin things by teleporting the player back to 0,0,0 instead of resetting the camera view like intended.